### ISSUE TITLE
Remove unnecessary place_goods state after choose planet

### ISSUE THREAD

**Issue by pickardj79** [pickardj79/bga-galaxytrucker#22]
With the planets card, the player chooses which planet to land on, then the game changes to a place goods state. The additional backend state change is unnecessary (more complicated) and prevents the player from being able to undo their planet choice.

With this example of client-side states, look for other places where they can be used to simplify the code and improve player experience.

**Comment by kuwizard** [comment]
I propose to look for other similar places in a separate ticket as it looks like a pretty much different task

**Comment by pickardj79** [comment]
Agreed, though maybe we don't need a separate ticket for it (yet) but just everyone keep an eye out for it? Especially once we have an example of how to do it implemented - which should be the outcome of this ticket.

Or... we could use this ticket as a holding place for a list of changes to make and then spin off tickets to do the work before this one closes?
